SOPT_27th_Android/app/src/main/java/sopt/onsopt/semina/presentation/login/LoginActivity.kt

```diff
@@ -44,16 +46,27 @@ class LoginActivity : AppCompatActivity() {
 
     private fun onLoginEvent() {
         if (loginViewModel.validateLoginForm()) {
-            changeMainActivityAndFinish()
-            showToast("Hello!!")
+            requestSignIn()
             return
         }
         showToast("check your id/pw")
     }
 
+    private fun requestSignIn() {
+        val signInData = SignInDomain(
+            email = loginViewModel.editTextId.get()!!,
+            password = loginViewModel.editTextPassword.get()!!
```

Comment:
```suggestion
            email = loginViewModel.editTextId.get()?: "이메일이 없습니다",
            password = loginViewModel.editTextPassword.get()?: "패스워드가 없습니다"
```
해당 부분이 Null이 안들어오는게 맞을까요?? 해당 부분에서 Null이 들어올 가능성이 많이 보여서.. !! 처리보다는 널처리를 해주시는 부분이 더 안정성이 있다 생각합니다!